### PR TITLE
swarm: release v0.4.3

### PR DESCRIPTION
**Pull request by skylenet**

### CODE REVIEW

**Review by nonsense — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
+## v0.4.3 (July 25, 2019)
 
 ### Notes
 
-### Features
-
-### Improvements
-
-### Bug fixes
+- **Docker users:** The `$PASSWORD` and `$DATA_DIR` environment variables are not supported anymore since this release. From now on you should mount the password or data dirctories as a volume. For example:
```
Comment (nonsense) [inline CHANGELOG.md]: I think the env var was called $DATADIR, not $DATA_DIR.

File: version/version.go [hunk 2]
```diff
@@ -21,10 +21,10 @@ import (
...
 
 const (
-	VersionMajor = 0          // Major version component of the current release
-	VersionMinor = 4          // Minor version component of the current release
-	VersionPatch = 3          // Patch version component of the current release
-	VersionMeta  = "unstable" // Version metadata to append to the version string
+	VersionMajor = 0        // Major version component of the current release
+	VersionMinor = 4        // Minor version component of the current release
+	VersionPatch = 3        // Patch version component of the current release
+	VersionMeta  = "stable" // Version metadata to append to the version string
```
Comment (nonsense) [inline version/version.go:27]: We should really get rid fo this at some point :)

**Review by nonsense — APPROVED** [review]

**Review by janos — APPROVED** [review]
LGTM

**Pull request merged**